fgd/bases/Weapon.fgd

```diff
@@ -6,15 +6,15 @@
 		[
 		1: "Start constrained" : 0
 		2: "Deny player pickup (reserve for NPC)" : 0 [-MOMENTUM]
-		4: "Not puntable by Gravity Gun" : 0 [HL2_ENTITIES]
+		4: "Not puntable by Gravity Gun" : 0 [+HL2_ENTITIES]
 		]
 
 	// Outputs
 	output OnPlayerUse(void) : "Fires when the player +uses this weapon."
 	output OnPlayerPickup(void) : "Fires when the player picks up this weapon."
 	output OnNPCPickup[-MOMENTUM](void) : "Fires when an NPC picks up this weapon."
-	output OnCacheInteraction[HL2_ENTITIES](void) : "Fires when the player 'proves' they've found this weapon. " +
+	output OnCacheInteraction[+HL2_ENTITIES](void) : "Fires when the player 'proves' they've found this weapon. " +
 		"Fires on: Player Touch, +USE pickup, Physcannon pickup, Physcannon punt."
-	output OnCacheInteraction[-HL2_ENTITIES](void) : "Fires when the player 'proves' they've found this weapon. " +
+	output OnCacheInteraction[-+HL2_ENTITIES](void) : "Fires when the player 'proves' they've found this weapon. " +
```

Comment:
Is this correct here? The -+?

Reply:
Nope! good catch, though im wondering why this sort of thing doesnt throw an error in srctools.